src/overlays/actors/ovl_En_Rg/z_en_rg.c

```diff
@@ -750,7 +750,7 @@ void EnRg_Init(Actor* thisx, PlayState* play) {
         this->actor.flags &= ~ACTOR_FLAG_1;
         this->unk_310 = 8;
         this->actor.gravity = -1.0f;
-        SubS_UpdateFlags(&this->unk_310, 3, 7);
+        SubS_UpdateFlags(&this->unk_310, SUBS_OFFER_MODE_ONSCREEN, SUBS_OFFER_MODE_MAX);
```

Comment:
Er, it doesn't look like this actor used one of the documented Subs functions, and the UpdateFlags function doesn't seem to be specific to any system, are you sure these are the right enums for this actor's stateFlag variable?

Reply:
Ah your right, I'll change it back. Though it is weird since I actually don't see those 3 bits getting used in `EnRg` (or `EnMtTag`) though maybe some other actor uses it 🤷.

Reply:
Because bits 1,2,4 are never used, that seems a bit too suspicious that those flags were still tied to the SUBS_OFFER modes and that it's simply doesn't use that feature. I would still tie this to the SUBS_OFFER since this seems like the only exception where `SubS_UpdateFlags` seems designed for the SUBS_OFFER? And then just add a comment noting these flags are unused. I would even still name `SubS_UpdateFlags` -> `SubS_UpdateOfferFlags` too.

Overall thoughts?

Reply:
Yeah I would be fine with it so long as there is documentation supporting the suspicion.

Reply:
Added a comment above, let me know if it isn't enough.